### DISCUSSION TITLE
"unstable_getServerSession" cookies(...).getAll is not a function

### DISCUSSION THREAD

**Discussion by pbl-js** [nextauthjs/next-auth#5859] [Help]
I've got Error: cookies(...).getAll is not a function when i use unstable_getServerSession() in server component.
Anyone had similar issue? 
```
import React from "react";
import "./globals.css";

import { Montserrat } from "@next/font/google";
import { ClientLayout } from "./ClientLayout";
import { unstable_getServerSession } from "next-auth";

const montserrat = Montserrat();

export default async function RootLayout({
  children,
}: {
  children: React.ReactNode;
}) {
  const session = await unstable_getServerSession();

  return (
    <html lang="en">
      <body className={montserrat.className}>
        <ClientLayout session={session}>{children}</ClientLayout>
      </body>
    </html>
```

**Comment by VaquarShaikh** [comment]
Hey , i had the same issue what machine are you using ?

**Reply by pbl-js** [reply]
I am using mac

**Comment by VaquarShaikh** [comment]
I was using windows , then i used wsl to try out , i reffered to this repo 
https://github.com/shadcn/taxonomy

this is a very helpful repo , i referred this and i dont have that error now

**Reply by pbl-js** [reply]
WOW. Looks super helpful. Thanks. I am checking if it solves my problem

**Comment by VaquarShaikh** [comment]
It later on worked both in windows and wsl for some reason tho

**Comment by pbl-js** [comment]
Suddenly I still have this issue ;C
